### DISCUSSION TITLE
SEQR local installation results show missing features

### DISCUSSION THREAD

**Discussion by samarthaFFM** [broadinstitute/seqr#2756] [Q&A]
**Describe the bug**
We have the latest version of SEQR installed locally. This version shows results without gene names and OMIM information on the results page as in the previous versions of SEQR. Is this to be expected because we run the default docker based pipeline-runner tool.

**Scope of the bug**
In all families and results

**Screenshots**
<img width="1419" alt="image" src="https://user-images.githubusercontent.com/42323324/170388284-bae6f755-5cd1-4ac9-bfb9-b4401d88f40c.png">

**Comment by hanars** [comment]
There appears to have been an error installing the reference data. This should happen automatically when you start the container, but you can try re-installing it with the following command:

```
docker-compose exec seqr python manage.py migrate --database=reference_data
docker-compose exec seqr python manage.py update_all_reference_data --use-cached-omim
```
